src/metalperformanceshaders.cs

```diff
@@ -131,25 +159,35 @@ interface MPSImagePyramid {
 		nuint KernelWidth { get; }
 	}
 
-	[iOS (10,0)][TV (10,0)]
+	[iOS (10,0)][TV (10,0)][Mac (10, 13, onlyOn64: true)]
 	[BaseType (typeof (MPSImagePyramid))]
 	[DisableDefaultCtor]
 	interface MPSImageGaussianPyramid {
 
 		// inlining .ctor from base class
 
 		[Export ("initWithDevice:kernelWidth:kernelHeight:weights:")]
-		[DesignatedInitializer]
 		[Internal]
-		IntPtr Constructor (IMTLDevice device, nuint kernelWidth, nuint kernelHeight, /* float* */ IntPtr kernelWeights);
+		IntPtr initWithDevice (IMTLDevice device, nuint kernelWidth, nuint kernelHeight, /* float* */ IntPtr kernelWeights);
```

Comment:
Should be `InitWithDevice`, capital `I`.

Weird so https://github.com/xamarin/xamarin-macios/pull/3005/files#diff-0a64265d7ab88b8af20f56544441b28aR249 is calling the parent right now?